prov/efa/src/rxr/rxr_ep.c

```diff
@@ -2596,13 +2651,17 @@ int rxr_endpoint(struct fid_domain *domain, struct fi_info *info,
 	rxr_ep->efa_outstanding_tx_ops = 0;
 	rxr_ep->shm_outstanding_tx_ops = 0;
 
-	ret = efa_cq_open(&efa_domain->util_domain.domain_fid, &cq_attr,
-			  &rxr_ep->rdm_cq, rxr_ep);
-	if (ret)
+	assert(!(rxr_ep->ibv_cq_ex));
```

Comment:
do you really need extra `()` around `rxr_ep->ibv_cq_ex`? especially you did not have it for L2658

Reply:
fixed